### ISSUE TITLE
sitewide 508-defect-1 ❗ Launchblocker [SCREENREADER]: role="alert" MUST not be used for a note

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#11909]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

```diff
! Team affected: VSA Authenticated Experience; Project: Profile 2.0
```

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

When the Direct Deposit screen loads, the first thing the screen reader reads aloud is:

> Note: If you think you’ve been the victim of bank fraud, please call us at 800-827-1000 (711). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.

This is because `role="alert"` was applied to this area. 

## Point of Contact

**VFS Point of Contact:** _First name only. Delete if not applicable._

## Acceptance Criteria

As a screen reader user, I want to review the content of the page and alerts only for items of significant relevance.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Start screen reader
1. Logged in, enter https://staging.va.gov/profile/direct-deposit in browser
1. Verify that the Note about bank fraud is the first thing read

## Possible Fixes (optional)

**Current code**
```html
<div class="usa-alert usa-alert-undefined background-color-only vads-u-margin-y--2 medium-screen:vads-u-margin-y--4" role="alert">
  <div class="usa-alert-body">
    <div class="usa-alert-text">
      <strong>Note:</strong> If you think you’ve been the victim of bank fraud, please call us at <a href="tel:1-800-827-1000" aria-label="800. 8 2 7. 1000." title="Dial the telephone number 800-827-1000" class="no-wrap">800-827-1000</a> (<a class="no-wrap " href="tel:+1711" aria-label="TTY: 7 1 1.">711</a>). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.
    </div>
  </div>
</div>
```
**Potential fix**
```html
<div class="usa-alert usa-alert-undefined background-color-only vads-u-margin-y--2 medium-screen:vads-u-margin-y--4">
  <div class="usa-alert-body">
    <div class="usa-alert-text">
      <strong>Note:</strong> If you think you’ve been the victim of bank fraud, please call us at <a href="tel:1-800-827-1000" aria-label="800. 8 2 7. 1000." title="Dial the telephone number 800-827-1000" class="no-wrap">800-827-1000</a> (<a class="no-wrap " href="tel:+1711" aria-label="TTY: 7 1 1.">711</a>). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.
    </div>
  </div>
</div>
```

## WCAG or Vendor Guidance (optional)

* [MDN Using the alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role)

**Comment by 1Copenut** [comment]
Agreed, this is not the first thing we want announced to users. It's important information to have on the screen, but sends the wrong message when it breaks into the normal reading flow.

**Comment by erikphansen** [comment]
Yeah this is definitely due to the change to AlertBox we've been talking about.

**Comment by Samara-Strauss** [comment]
Related to #11910. Is it safe to assume these will be resolved at the same time?

**Comment by erikphansen** [comment]
[This PR reverts the change](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/363). When that is merged and published we'll need to update `vets-website` to use it.

**Comment by mshea0606** [comment]
This ticket is for the platform team. Our team does not have work to do for this ticket.

**Comment by jenstrickland** [comment]
I adjusted the tags and title, assigned to the frontend team to triage, and assigned @1Copenut since he's on top of getting that other issue rolled back.

**Comment by 1Copenut** [comment]
@jenstrickland This work is underway in the design system package here: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/363/files

**Comment by erikphansen** [comment]
This'll be fixed with [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/13795)

**Comment by erikphansen** [comment]
@jenstrickland this should be fixed in staging now. Please validate and close at your leisure.

**Comment by jenstrickland** [comment]
Validated! 🎉
